### PR TITLE
refactor(traffic): moving instant traffic to another section

### PR DESCRIPTION
**Pull request by Tbaile**
Closes: https://github.com/NethServer/nethsecurity/issues/936

--- 

Screenshots:

### CODE REVIEW

**Comment by Tbaile** [comment]
Just noticed that `Filter` section shouldn't be the size of the table. However I feel this is more "correct" way instead of cropping the field to half the size. Asking review to Lucia to get some feedback.

**Comment by Tbaile** [comment]
Also, for @andre8244, the current cap of total host per list is 20, shall we remove the limit?

**Comment by andre8244** [comment]
> Also, for @andre8244, the current cap of total host per list is 20, shall we remove the limit?

Don't know, you can ask in the team chat if you want

File: src/components/standalone/monitoring/traffic/InstantAppTrafficCard.vue [hunk 1]
```diff
@@ -41,12 +51,14 @@ function formatTraffic(value: number) {
 </script>
 
 <template>
-  <NeCard
-    :title="t('standalone.real_time_monitor.applications')"
-    :skeletonLines="5"
-    :errorTitle="error"
-    :errorDescription="errorDescription"
-  >
+  <NeCard :skeletonLines="5" :title="t('standalone.real_time_monitor.applications')">
```
Comment (andre8244) [inline src/components/standalone/monitoring/traffic/InstantAppTrafficCard.vue:54]:
```suggestion
  <NeCard :skeletonLines="5" :loading="loading" :title="t('standalone.real_time_monitor.applications')">
```

File: src/components/standalone/monitoring/traffic/InstantHostTrafficCard.vue [hunk 2]
```diff
@@ -41,12 +51,14 @@ function formatTraffic(value: number) {
 </script>
 
 <template>
-  <NeCard
-    :title="t('standalone.real_time_monitor.local_hosts')"
-    :skeletonLines="5"
-    :errorTitle="error"
-    :errorDescription="errorDescription"
-  >
+  <NeCard :skeletonLines="5" :title="t('standalone.real_time_monitor.local_hosts')">
```
Comment (andre8244) [inline src/components/standalone/monitoring/traffic/InstantHostTrafficCard.vue:54]:
```suggestion
  <NeCard :skeletonLines="5" :loading="loading" :title="t('standalone.real_time_monitor.local_hosts')">
```

File: src/components/standalone/monitoring/traffic/InstantProtocolTrafficCard.vue [hunk 3]
```diff
@@ -41,12 +51,14 @@ function formatTraffic(value: number) {
 </script>
 
 <template>
-  <NeCard
-    :title="t('standalone.real_time_monitor.protocols')"
-    :skeletonLines="5"
-    :errorTitle="error"
-    :errorDescription="errorDescription"
-  >
+  <NeCard :skeletonLines="5" :title="t('standalone.real_time_monitor.protocols')">
```
Comment (andre8244) [inline src/components/standalone/monitoring/traffic/InstantProtocolTrafficCard.vue:54]:
```suggestion
  <NeCard :skeletonLines="5" :loading="loading" :title="t('standalone.real_time_monitor.protocols')">
```

**Comment by Tbaile** [comment]
Why the need to handle loading for each card? Isn't better to put everything [behind the skeleton](https://github.com/NethServer/nethsecurity-ui/pull/446/files#diff-76b2c37ff2ba3948e2e470cd944f0bf8f56bd4c32d80a88eda7819fb7a95083dR19)?

**Pull request merged**